plugins/woocommerce/includes/log-handlers/class-wc-log-handler-db.php

```diff
@@ -78,7 +78,11 @@ protected static function add( $timestamp, $level, $message, $source, $context )
 		);
 
 		if ( ! empty( $context ) ) {
-			$insert['context'] = serialize( $context ); // @codingStandardsIgnoreLine.
+			try {
+				$insert['context'] = serialize( $context ); // @codingStandardsIgnoreLine.
+			} catch ( Exception $e ) {
+				$insert['context'] = 'There was an error while serializing the context: ' . $e->getMessage();
```

Comment:
this error message needs to be serialized before storing otherwise it will trip when reading from this log

Reply:
You are right. Solved.